rules/Php81/Rector/MethodCall/MyCLabsMethodCallToEnumConstRector.php

```diff
@@ -111,17 +120,19 @@ private function refactorGetValueMethodCall(MethodCall $methodCall): ?PropertyFe
         }
 
         $staticCall = $methodCall->var;
-        $className = $this->getName($staticCall->class);
-        if ($className === null) {
+        $originalName = $staticCall->class->getAttribute(AttributeKey::ORIGINAL_NAME);
```

Comment:
This should be untouched as other rules always work with fun class names.
Making it short could confuse them and work incorrectly.

Reply:
This one needs to be resolved before merge.